src/qmxgraph/exceptions.py

```diff
@@ -0,0 +1,17 @@
+class QmxGraphError(Exception):
+    """
+    Base exception for all exceptions raised by qmxgraph.
+    """
+
+
+class InvalidJavaScriptError(QmxGraphError):
```

Comment:
Could you check if this class is being used?

Reply:
Good eye my man 😉 

Not in this one, but will be used in my next PR. 👍